src/QsCompiler/Core/Transformations/SyntaxTreeTransformation.fs

```diff
@@ -262,6 +263,7 @@ type SyntaxTreeTransformation(options) as this =
 
     /// Invokes the transformation for all namespaces in the given compilation.
     abstract OnCompilation: compilation: QsCompilation -> QsCompilation
+
```

Comment:
This is from me running Fantomas, for the record. (I am not losing my mind and adding empty lines to random files)

Reply:
Usually that means your Fantomas version is different from what the repo specifies? There are only a few extra changes though so it's not a big deal.

Reply:
After I run `dotnet tool restore`, I get the following version of Fantomas:

```
$ dotnet tool run fantomas --version
Fantomas v4.7.9+cbaa31cc6977666ab1103b1ee49aff31b85e39d8
```

That seems to match the 4.7.9 version in `.config/dotnet-tools.json`, but I am just a humble Boy Scout